netlify/functions/leaderboard.ts

```diff
@@ -23,7 +24,7 @@ const getLeaderboardResults = async (
   handle?: string
 ) => {
   // @TODO: also filter by contestId (if provided)
-  const allContests = (await csv().fromFile("_data/contests/contests.csv"))
+  const allContests = (await getApiContestData())
```

Comment:
See here we are making the API call instead of using the graph for example, so I am not sure we need to feed the graph.